assignments/arrays.js

```diff
@@ -63,34 +63,54 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}` );
 
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
 let lastCar = 0;
```

Comment:
They were expecting you to set `lastCar` to `inventory[inventory.length-1]`, but the way you did it is fine, too. I did it the same way you did, and I think that writing out the whole chain is a good way to reinforce the underlying structure. That said, in production-level code there are a lot of good reasons to use variables to represent values you're going to use repeatedly.